vespajlib/src/main/java/com/yahoo/tensor/serialization/DenseBinaryFormat.java

```diff
@@ -66,6 +68,16 @@ private void encodeFloatCells(IndexedTensor tensor, GrowableByteBuffer buffer) {
             buffer.putFloat(tensor.getFloat(i));
     }
 
+    private void encodeBFloat16Cells(IndexedTensor tensor, GrowableByteBuffer buffer) {
+        for (int i = 0; i < tensor.size(); i++)
+            buffer.putShort((short)(Float.floatToRawIntBits(tensor.getFloat(i)) >>> 16));
```

Comment:
consider adding some utility methods (in TypedBinaryFormat maybe) for the low-level bit shuffling:
```
    static short bitsFromBFloat16(float bf16) {
        int bits = Float.floatToRawIntBits(bf16);
        return (short) (bits >>> 16);
    }

    static float bFloat16FromBits(short bits) {
        return Float.intBitsToFloat(bits << 16);
    }
 ```

Reply:
Thanks, fixed.